scripts/migration/migrate_deleted_wikis.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from website.app import init_app
+from website.models import NodeLog
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+def get_targets():
+    return NodeLog.find(Q('action', 'eq', NodeLog.WIKI_DELETED))
+
+def migrate(targets, dry_run=True):
+    # iterate over targets
+    logs = targets
+    nodes = set()
+    for log in logs:
+        nodes.add(log.node)
+    for node in nodes:
+        versions = node.wiki_pages_versions
+        current = node.wiki_pages_current
+        updated_versions = {}
+        for wiki in node.wiki_pages_versions:
```

Comment:
Since you've saved `node.wiki_pages_versions` to a `versions`, go ahead and use `versions` here.